curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/implement-a-root-level-request-logger-middleware.english.md

```diff
@@ -6,7 +6,7 @@ challengeType: 2
 
 ## Description
 <section id='description'>
-Before we introduced the <code>express.static()</code> middleware function. Now it’s time to see what middleware is, in more detail. Middleware functions are functions that take 3 arguments: the request object, the response object, and the next function in the application’s request-response cycle. These functions execute some code that can have side effects on the app, and usually add informations to the request or response objects. They can also end the cycle sending the response, when some condition is met. If they don’t send the response, when they are done they start the execution of the next function in the stack. This is triggered calling the 3rd argument <code>next()</code>. More information in the <a href='http://expressjs.com/en/guide/using-middleware.html' target='_blank'>express documentation</a>.
+Before we introduced the <code>express.static()</code> middleware function. Now it’s time to see what middleware is, in more detail. Middleware functions are functions that take 3 arguments: the request object, the response object, and the next function in the application’s request-response cycle. These functions execute some code that can have side effects on the app, and usually add information to the request or response objects. They can also end the cycle sending the response, when some condition is met. If they don’t send the response, when they are done they start the execution of the next function in the stack. This is triggered calling the 3rd argument <code>next()</code>. More information in the <a href='http://expressjs.com/en/guide/using-middleware.html' target='_blank'>express documentation</a>.
```

Comment:
```suggestion
Before, we had introduced the <code>express.static()</code> middleware function. Now it’s time to see what middleware is in more detail. Middleware functions are functions that take 3 arguments: the request object, the response object, and the next function in the application’s request-response cycle. These functions execute some code that can have side effects on the app, and usually add information to the request or response objects. They can also end the cycle by sending the response, when some condition is met. If they don’t send the response, when they are done they start the execution of the next function in the stack. This is triggered by calling the 3rd argument <code>next()</code>. More information in the <a href='http://expressjs.com/en/guide/using-middleware.html' target='_blank'>express documentation</a>.
```

Reply:
I think `Before, we had introduced` should be `Earlier, we introduced`

Reply:
Agree with @RandellDawson. I had not thoroughly reviewed the text. Just tried to fix something that stood out. But now that @thecodingaviator surfaced it, I believe "Earlier" is clearer than "Before" in this context.

Who adds that change? I'm not fully up to speed on FCC Github etiquette.

Reply:
@YiddishNinja It is your PR, so you would make the change.  But before you do, let's make sure @thecodingaviator is on board with the idea.

Reply:
I'm good with that too @RandellDawson . Sorry for the late reply.

Also Randell, I guess it's fine if @YiddishNinja makes the change